File: react-native-app/android/app/src/main/java/com/fitsnitchapp/api/CreateSnitchRequest.java
package com.fitsnitchapp.api;

import com.fitsnitchapp.LatLonPair;
import com.fitsnitchapp.Restaurant;

public class CreateSnitchRequest {
    public String userId;
    public LatLonPair originCoords;
    public Restaurant restaurantData;

    public CreateSnitchRequest(String userId, LatLonPair coords, Restaurant restaurantData) {
        this.userId = userId;
        this.originCoords = coords;
        this.restaurantData = restaurantData;
    }
}

File: new-react-app/shared/models/requests/CreateSnitchRequest.ts
import { LatLonPair } from "../CoordinateModels";
import RestaurantData from "../RestaurantData";

export class CreateSnitchRequest {
    constructor(
        public userId:string,
        public restaurantData:RestaurantData,
        public originCoords:LatLonPair
    ) {}
}
